src/main/java/bio/overture/ego/model/dto/VisaUpdateRequest.java

```diff
@@ -0,0 +1,25 @@
+package bio.overture.ego.model.dto;
+
+import java.util.UUID;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class VisaUpdateRequest {
+
+  @NotNull private UUID id;
```

Comment:
should we be able to update Visa id? if not, we can remove id field.

Reply:
Removed UUID from update request structure